e2e/suite1/connections-actions.test.ts

```diff
@@ -0,0 +1,295 @@
+import * as connections from '../helpers/connection-actions';
+import disableTransitions from '../helpers/disableTransitions';
+import insertFixtures from '../helpers/insertFixtures';
+import { adminLogin, logout } from '../helpers/login';
+import proxyMock from '../helpers/proxyMock';
+
+describe('connections', () => {
+  beforeAll(async () => {
+    await insertFixtures();
+    await proxyMock();
+    await adminLogin();
+    await disableTransitions();
+  });
+
+  afterAll(async () => {
+    await logout();
+  });
+
+  it('should navigate to an entities connections view', async () => {
```

Comment:
this should be `entity's connections view`, not `entities`.

Reply:
Done.